lib/protractor-testability-plugin.js

```diff
@@ -22,8 +22,24 @@ return {
                         };
                     },
                     getTestability: function () {
+                        // Allow users to set their own testability objects to
+                        // map anything Angular provides for Protractor to the
+                        // framework they're using.
+                        var whenStableDefault = function (cb) { cb(); };
+                        var customAngularTestability = window.customAngularTestability;
+                        if (customAngularTestability) {
+                            // Allow setting custom whenStable of any value if
+                            // it's assigned. If not assigned, use default.
+                            // The user may want to set whenStable to anything,
+                            // even falsey, for testing reasons, so they should
+                            // be allowed to do so.
+                            if (typeof customAngularTestability.whenStable === 'undefined') {
+                                customAngularTestability.whenStable = whenStableDefault;
+                            }
+                            return customAngularTestability;
+                        }
                         return {
-                            whenStable: function (cb) { cb(); }
+                            whenStable: whenStableDefault
```

Comment:
Mmmm.... I don't quite understand. I think the snippet I wrote does exactly the same than the one in the PR (lines 25-42), right? Did I miss some use case? I mean, the snippet will return an object with the `whenStable` equal to the default unless `customAngularTestability` has one on it's own, plus the returned object will contain all the properties included in `customAngularTestability`.